README.md

```diff
@@ -58,34 +58,34 @@ The following figure shows the typical CoatRack architecture, the CoatRack web a
 
 
 
-## Installation using production-ready Docker Images
+## Installation using Docker Images
 
  The following prerequisites are required:
 
-*  a Linux shell - Windows users can use WSL of the Git Bash shell instead
+*  a Linux shell - Windows users can use WSL or the Git Bash shell instead
 *  docker
 *  docker-compose
 
 
 
-### Deployment For Newcomers
+### First Example Deployment
 
-This approach is suggested if you are completely new to CoatRack and want to just want to experience what it actually does. Follow these instructions:
+This approach is suggested if you are familiar with the basic CoatRack functionality provided by [coatrack.eu](https://coatrack.eu) and you want to deploy your own CoatRack instance. Go to the directory `docker/docker-compose-deployment` and follow these instructions:
 
-1. Change the `INSERT_SAMPLE_DATA_ON_STARTUP` parameter in `.env` to `true` to initialize the example Gateway in the database.
-2. Go to the directory `docker/docker-compose-deployment` and execute:
+1. Change the `INSERT_SAMPLE_DATA_ON_STARTUP` parameter in `.env` to `true` to initialize some example data. This step is optional.
```

Comment:
If the sample data is not included, the access to the sample service via the sample gateway will probably not work. So I am not sure if this makes sense.

Reply:
You are correct which is why I corrected that. I also introduced a simple bash script which can be run, to set up the example instance of CoatRack using example data and an example gateway.